eras/alonzo/test-suite/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-alonzo-test`
 
+## 1.2.2.0
+
+* Require `cardano-ledger-core` >= 1.15
+
```

Comment:
If you look at the PR template it will say in bold: new sections are never added to changelogs. It is only when we do a post-release PR that we add new sections for all packages that have been released
```suggestion
```